specification/security/resource-manager/Microsoft.Security/preview/2023-07-01-preview/integrations.json

```diff
@@ -0,0 +1,653 @@
+          "description": "Integration creation date",
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true
+        },
+        "updatedBy": {
+          "description": "Integration last updated by object id (GUID)",
+          "type": "string",
+          "readOnly": true
+        },
+        "updatedOn": {
+          "description": "Integration last update date",
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true
+        }
+      }
+    },
+    "IntegrationAdditionalData": {
+      "type": "object",
+      "description": "The integration additional data (abstract)",
+      "discriminator": "integrationPlatform",
+      "properties": {
+        "integrationPlatform": {
+          "type": "string",
+          "description": "Describes the integration platform"
+        }
+      },
+      "required": [
+        "integrationPlatform"
+      ]
+    },
+    "ServiceNowAdditionalData": {
+      "description": "Service Now specific additional data",
+      "allOf": [
+        {
+          "$ref": "#/definitions/IntegrationAdditionalData"
+        },
+        {
+          "properties": {
+            "instanceUrl": {
+              "description": "Integration creation date",
+              "type": "string"
+            },
+            "clientId": {
+              "description": "Integration creation date",
+              "type": "string"
+            },
+            "clientSecret": {
+              "description": "Integration creation date",
+              "type": "string",
+              "x-ms-secret": true
+            },
+            "tableMetadata": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/ServiceNowTableMetadata"
+              }
+            }
+          }
+        }
+      ],
+      "x-ms-discriminator-value": "ServiceNow"
+    },
+    "ServiceNowAuthenticationDetails": {
+      "description": "Service Now authentication details",
+      "type": "object",
+      "properties": {
+        "userName": {
+          "type": "string",
+          "description": "The integration user name"
+        },
+        "password": {
+          "type": "string",
+          "description": "The integration user password",
+          "x-ms-secret": true
+        },
+        "clientId": {
+          "type": "string",
+          "description": "The integration client id"
+        },
+        "clientSecret": {
+          "type": "string",
+          "description": "The integration client secret",
+          "x-ms-secret": true
+        },
+        "instanceUrl": {
+          "type": "string",
+          "description": "The integration instance url"
+        }
+      }
+    },
+    "ServiceNowMetadataResponse": {
+      "description": "Describes the Service Now authentication response and if successful, contains tables metadata response",
+      "type": "object",
+      "properties": {
+        "authenticationSucceeded": {
```

Comment:
>authenticationSucceeded [](http://example.com/codeflow?start=9&length=23)

Boolean?

Reply:
Changd authenticationSucceeded to bool - done.